### ISSUE TITLE
Feature Request - pool / solo switch

### ISSUE THREAD

**Issue by Maveth** [OCEAN-xyz/datum-gateway-startos#1]
Feature Request

In order to switch easily from a pool to solo mining, the fields need to be blanked out, then re-entered to switch back.
It would be nice to have a toggle switch, rather than have to go find the info and fill it out each time.
solo = true (send blank info) - pool = false (send the data in the fields) - or the opposite way.
But the info can be left in place, the toggle switch just overrides the data when enabled.

**Comment by BitcoinMechanic** [comment]
Other pools can support DATUM too so it isn't correct to equate pooled mining with DATUM with using OCEAN or any particular pool.

**Comment by BitcoinMechanic** [comment]
This has been changed a little in the latest release, but still isn't what you're requesting which I don't think is going to be doable.

**Comment by Maveth** [comment]
Its not really, a pool specific issue.
Its more a startOS UI QOL thing.
I would like to leave my pool info (currently pointed to ocean) in the UI, but have a toggle switch that ignores the info so that it switches to solo, without losing the settings for my default pool.

toggle:
Solo <> Pool
When on solo it just ignores the pool info and solo mines, when on pool it takes the pool info and sends there.

Its not a huge issue, but every once in a while the mood strikes and I like to play the lotto for a little bit, copy and pasting the info back into the fields is fine.

**Comment by BitcoinMechanic** [comment]
Yeah I understand. We're reworking how the GUI config in StartOS works anyway but have to get familiar with incoming 0.3.6 before making any promises.

**Comment by Retropex** [comment]
Can be done with the `reward_sharing` option in the UI.
